cmd/xcodeUITests.go

```diff
@@ -0,0 +1,134 @@
+package cmd
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/colorstring"
+	"github.com/bitrise-io/go-utils/fileutil"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/goinp/goinp"
+	"github.com/bitrise-tools/codesigndoc/codesigndocuitests"
+	"github.com/bitrise-tools/codesigndoc/xcodeuitest"
+	"github.com/bitrise-tools/go-xcode/utility"
+	"github.com/spf13/cobra"
+)
+
+var xcodeUITestsCmd = &cobra.Command{
+	Use:   "xcodeuitests",
+	Short: "Xcode project scanner for UI tests",
+	Long:  `Scan an Xcode project for UI test targets`,
+
+	SilenceUsage:  true,
+	SilenceErrors: true,
+	RunE:          scanXcodeUITestsProject,
+}
+
+func init() {
+	scanCmd.AddCommand(xcodeUITestsCmd)
+
+	xcodeUITestsCmd.Flags().StringVar(&paramXcodeProjectFilePath, "file", "", "Xcode Project/Workspace file path")
+	xcodeUITestsCmd.Flags().StringVar(&paramXcodeScheme, "scheme", "", "Xcode Scheme")
```

Comment:
also, I don't agree with this solution. It is because we wanted a tool that if became installed on the machine then if the user cd-s into an xcode project dir then we should fetch all infos that possible for example if we find an xcworkspace and/or project in that dir then use that, if we find only one scheme then use that etc, these flags should be acceptable for only manual config, otherwise find out as much as we can, and if there is more than one option then ask them from the user, so if we found 3 schemes, ask them to input which one.